pyproject.toml

```
@@ -12,6 +12,15 @@ great-expectations = "^0.17.17"
 pydantic = "<3"
 pika = "^1.3.1"
 sqlalchemy = ">=1.4.49"
+snowflake-connector-python = [
+    {version = ">=2.5.0", python = "<3.11", optional = true},
+    {version = ">2.9.0", python = ">=3.11", optional = true}
+]
+snowflake-sqlalchemy = { version = ">=1.2.3", optional = true }
```

```suggestion
snowflake-sqlalchemy = { version = ">=1.5.0", optional = true }
```
We don't need to be so broadly compatible for the agent.